transfer_learning.ipynb

```diff
@@ -289,11 +270,7 @@
     "\n",
     "        # Each epoch has a training and validation phase\n",
     "        for phase in ['train', 'val']:\n",
-    "            if phase == 'train':\n",
-    "                scheduler.step()\n",
-    "                model.train()  # Set model to training mode\n",
-    "            else:\n",
-    "                model.eval()   # Set model to evaluate mode\n",
+    "            model.train(phase == 'train')  # Set model to training / eval mode\n",
```

Comment:
Feel free to revert this change if you think it was clearer before (but remove the `scheduler.step()`, as I mentioned in the other comment